dcaf_case__management/app/views/case/index.html.erb

```diff
@@ -0,0 +1,97 @@
+<div class="container">
```

Comment:
YES. Absolutely love that you started to work in the early designs. Thanks **SO** much for doing that.